tests/unit/LatinCharValidatorTest.php

```diff
@@ -54,5 +54,9 @@ public function testIsValid()
         $this->assertFalse(LatinCharValidator::isValid('الْأَبْجَدِيَّة الْعَرَبِيَّة‎'));
         // Russian chars
         $this->assertFalse(LatinCharValidator::isValid('Меня зовут Мандли'));
+        // moji
+        $this->assertFalse(LatinCharValidator::isValid('Hello World 😀'));
```

Comment:
isValid ?

Reply:
And we can also split into 2 validator so we can use it in the different context separaltly if we want